burnman/composite.py

```diff
@@ -199,6 +199,13 @@ def to_string(self):
         """
         return "{0}: {1}".format(self.__class__.__name__, self.name)
 
+    @material_property
+    def formula(self):
+        """
+        Returns molar chemical formula of the composite
+        """
+        return sum_formulae([ph.formula for ph in self.phases], self.molar_fractions)
```

Comment:
what is ``formula`` doesn't exist? Maybe we can produce a better error message?

Reply:
There are already error messages written for solidsolution and mineral, which pop up either during init (solidsolution) or when formula is called (mineral). Is there anywhere else we want errors?

Reply:
ah, yes. You are right. I didn't see that ``.formula`` here is a function call (and it will give an error for minerals without a formula).